rpc/json/ws.go

```diff
@@ -68,7 +68,8 @@ func (h *handler) wsHandler(w http.ResponseWriter, r *http.Request) {
 		}
 
 		if mt != websocket.TextMessage {
-			h.logger.Error("expected text message")
+			// TODO(tzdybal): https://github.com/celestiaorg/optimint/issues/465`
```

Comment:
This looks orthogonal to this PR, can you do it in a separate PR?

Reply:
https://github.com/celestiaorg/optimint/pull/469